damus Localizations/en-US.xcloc/Localized Contents/en-US.xliff

```diff
@@ -285,6 +285,16 @@ Number of profiles a user is following.</note>
         <target>DM</target>
         <note>Navigation title for DM view, which is the English abbreviation for Direct Message.</note>
       </trans-unit>
+      <trans-unit id="DM Type" xml:space="preserve">
```

Comment:
@jb55 Anytime we change strings, we need to go to Xcode and run `Product` -> `Export Localizations...` and select `English (United States)` only (no other languages should / need to be checked off) so that it can run through the translation pipeline.

I'll make this automated soon, but fyi in the meantime.

Reply:
hmm ok good to know